src/Cake.Common/Tools/OpenCover/OpenCoverSettings.cs

```diff
@@ -67,7 +67,7 @@ public sealed class OpenCoverSettings : ToolSettings
         public bool MergeOutput { get; set; }
 
         /// <summary>
-        /// Gets or sets a list of directories where assemblies being loaded from will be ignored.
+        /// Gets a list of directories where assemblies being loaded from will be ignored.
```

Comment:
For some reason, I got StyleCop errors for this file so I updated it. I have no clue why...